bundles/org.openhab.binding.matter/README.md

```diff
@@ -0,0 +1,472 @@
+
+The pairing action can be found in the settings of the "Controller" thing under the "Actions" -> "Pair Matter Device" 
+
+<img src="doc/pairing.png" alt="Matter Pairing" width="600"/>
+
+This action will give feedback on the pairing process, if successful a device will be added to the Inbox.
+
+See [Device Pairing: Code Types](#device-pairing-code-types) for more information on pairing codes and code formats.
+
+The same codes can also be used in the openHAB Thing discovery UI, although feedback is limited and only a single controller is supported.  
+
+<img src="doc/thing-discovery.png" alt="Thing Discovery" width="600"/>
+
+### Device Pairing: Code Types
+
+In order to pair (commission in matter terminology) a device, you must have an 11 digit manual pairing code (eg 123-4567-8901 or 12345678901) or a QR Code (eg MT:ABCDEF1234567890123).  If the device has not been paired before, use the code provided by the manufacturer and **ensure the device is in pairing mode**, refer to your devices instructions for pairing for more information. You can include dashes or omit them in a manual pairing code.
+
+If the device is paired with another Matter ecosystem (Apple, Google, Amazon, etc..) you must that ecosystem to generate a new pairing code and search for devices.  The pairing code and device will only be available for commissioning for a limited time.  Refer to the ecosystem that generated the code for the exact duration (typically 1-15 minutes). In this case, openHAB still talks directly to the device and is not associated with that existing ecosystem. 
+
+If the device seems to be found in the logs, but can not be added, its possible the device has been already paired.  Hard resetting the device may help this case.  See your device documentation for how to hard reset the device.
+
+### Device Pairing: Thread Devices
+
+Thread devices require a Thread Border Router and a bluetooth enabled device to facilitate the thread joining process (typically a mobile device).  Until there is a supported thread border router integration in openHAB and the openHAB mobile apps, it's strongly recommended to pair the device to a commercial router with thread support first (Apple TV 4k, Google Nest Hub 2, Amazon Gen 4 Echo, etc... ), then generate a matter pairing code and add the device normally.  This will still allow openHAB to have direct access to the device using only the embedded thread border router and does not interact with the underlying providers home automation stack.
+
+Support for using a Open Thread Border Router has been verified to work and will be coming soon to openHAB, but in some cases requires strong expertise in IPv6 routing as well as support in our mobile clients. 
+
+### Enabling IPv6 Thread Connectivity on Linux Hosts
+
+It is important to make sure that Route Announcements (RA) and Route Information Options (RIO) are enabled on your host so that Thread boarder routers can announce routes to the Thread network. 
```

Comment:
```suggestion
It is important to make sure that Route Announcements (RA) and Route Information Options (RIO) are enabled on your host so that Thread Border Routers can announce routes to the Thread network. 
```